R/epinowcast.R

```diff
@@ -190,7 +190,7 @@ epinowcast <- function(data,
     )
   )
 
-  out <- cbind(data, fit)
+  out <- cbind(data, fit, priors = list(priors))
```

Comment:
we might want priors to appear before the fit output for the flow to work?

Reply:
See change in e277b5b83ce09cf695e797cc413ffce324a9b7b1.